### DISCUSSION TITLE
Could you please put these games on friv reborn ?? Those were my favorite games!!

### DISCUSSION THREAD

**Discussion by frivclassic2022** [Tsyuay/FrivReborn#29] [Q&A] [answered]
•Noitcelfer. keyboard buttons: up, down, left, right > arrows, space, w s a d r.
•Checkpoint. keyboard buttons: up, down, left, right > arrows, space, w s a d p.
•Monkey Fu. keyboard buttons: up, down, left, right > arrows, space.

swf 👇

Noitcelfer: https://img-hws.y8.com/cloud/v2-y8-flash-games-001/contents/item_versions/flash_games/95859/original/noitcelfer.swf?1528860309"

Checkpoint: https://img-hws.y8.com/cloud/y8-flash-game/contents/item_versions/flash_games/21439/original/checkpoint.swf?1521374389"

Monkey Fu: https://cdn.game-game.com.ua/games/63695.swf

Please!! I played these games a lot!!! 🙏🙏🙏

**Comment by frivclassic2022** [comment]
Hello?

**Comment by Tsyuay** [comment, marked as answer]
done
